kesko/crates/kesko_models/src/urdf_model.rs

```diff
@@ -0,0 +1,343 @@
+                                    material: material.clone(),
+                                    mesh: meshes.add(
+                                        shape::Icosphere {
+                                            radius: radius as f32,
+                                            subdivisions: 5,
+                                        }
+                                        .try_into()
+                                        .unwrap(),
+                                    ),
+                                    transform,
+                                    ..default()
+                                });
+                            }
+                            urdf_rs::Geometry::Box { size } => {
+                                println!("box size: {:?}", size);
+                                commands.entity(visual_entity).insert(PbrBundle {
+                                    material: material.clone(),
+                                    mesh: meshes.add(
+                                        shape::Box {
+                                            min_x: -size.0[0] as f32 / 2.0,
+                                            min_y: -size.0[1] as f32 / 2.0,
+                                            min_z: -size.0[2] as f32 / 2.0,
+                                            max_x: size.0[0] as f32 / 2.0,
+                                            max_y: size.0[1] as f32 / 2.0,
+                                            max_z: size.0[2] as f32 / 2.0,
+                                        }
+                                        .try_into()
+                                        .unwrap(),
+                                    ),
+                                    transform,
+                                    ..default()
+                                });
+                            }
+                            urdf_rs::Geometry::Cylinder { radius, length } => {
+                                println!("cylinder radius: {}, length: {}", radius, length);
+                            }
+                            urdf_rs::Geometry::Capsule { radius, length } => {
+                                println!("capsule radius: {}, length: {}", radius, length);
+                            }
+                            urdf_rs::Geometry::Mesh {
+                                ref filename,
+                                scale,
+                            } => {
+                                // replace path prefix 
+                                let filename = filename.replace("package://crane_x7_description", "/home/azazdeaz/repos/art-e-fact/wizard_separate_tests/gen/crane_x7_test_project/src/crane_x7_description");
+                                let filename = filename.replace("package://sciurus17_description", "/home/azazdeaz/repos/art-e-fact/wizard_separate_tests/gen/crane_x7_test_project/src/sciurus17_ros/sciurus17_description");
```

Comment:
I suppose this is currently hardcoded for testing purpose.
How would you proceed to generate a "world", like gazebo? It would be Rust code? or could it be some description format, maybe even sdf?

Reply:
@azazdeaz By the way, if I want to see one of those urdf in the sim, I suppose I need to change the path in this line. Anything else to do?

Reply:
I think the best way to create a world is spawning objects with the Python API (which could be improved, maybe by taking lessions from Sapien) :
https://github.com/art-e-fact/kesko/pull/1/files#diff-f812d884d84336a552208d221a13fd73cbc4951d8c7aafb3d53bbbc82375b11aR15-R18

Yes, the URDF paths are currently hard coded, but i'll add a python API for that too :+1:

Reply:
> I think the best way to create a world is spawning objects with the Python AP

However, if we web-embed kesko, we will need to interface via js? unless we want to wasm the python API too?

> Yes, the URDF paths are currently hard coded, but i'll add a python API for that too 

nice

Reply:
We could interface via JS, but we can also run python on the server and let it communicate with Kesko via messages. Kesko currently has two transport option for the Python API. One is the direct bindings, and one is over TCP. I'm not sure if TCP works in the browser out of the box, but it shouldn't be hard to get it working over websockets ie.

Reply:
That's a good idea. I am wondering hw we would want to deal with authentication and such, but this can maybe come later